spyder/config/main.py

```diff
@@ -1,23 +1,25 @@
 # -*- coding: utf-8 -*-
-#
+# ----------------------------------------------------------------------------

```

Comment:
Please remove these separation lines. We decided not to use them at some point (probably CAM's idea).

Reply:
Will do!